user_tools/src/spark_rapids_pytools/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""init file of the spark_rapids_dataproc package."""
+
+from spark_rapids_pytools.build import get_version
+
+VERSION = '22.12.0'
```

Comment:
Is this supposed to remain 22.12.0? or should it be bumped to 23.02.0?

Reply:
Yes, I have updated the version after confirming with Matt.